source/content/guides/drupal-9/drupal-9-hosted-pre112021/10-deploy.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Migrate a Site That Was Created Before November 2011 to Drupal 9
+subtitle: Deploy
+description: 
+cms: "Drupal 9"
+categories: [develop]
+tags: [code, launch, migrate, site, updates]
+contributors: [wordsmither]
+reviewed: "2021-03-31"
+layout: guide
+permalink: docs/guides/drupal-9-hosted-pre112021/deploy
+anchorid: deploy
+editpath: drupal-9/drupal-9-hosted-pre112021/11-deploy.md
+---
+<Partial file="drupal-9/deploy-live.md" />
```

Comment:
This is just one line - "Review the site, deploy to Test, and finally to Live. You're done!" 
 Is it possible to add additional relevant content or link this page with the previous section? WDYT?

Reply:
I'm trying to follow a common structure for all guides - and every guide should have a "deploy" page.  I know it's just one line, but I still think it's worth keeping this way for consistency.